docs/index.rst

```diff
@@ -9,9 +9,9 @@
   AP LiquidHandle <liquid_handle>
   AP Other <autoprotocol>
   Changelog <changelog>
-  CONTRIBUTING
-  AUTHORS
-  LICENSE
+  Contributing <contributing>
```

Comment:
Huh! I'm impressed that this resolves properly!

Reply:
heh, yea there's a contributing.rst file included, which references the top-level field